include/zephyr/input/input_touch.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2024 Antmicro <www.antmicro.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_TOUCH_H_
+#define ZEPHYR_INCLUDE_TOUCH_H_
+
+/**
+ * @brief Touch Events API
+ * @defgroup touch_events Touchscreen Event Report API
+ * @since 3.7
+ * @version 0.1.0
+ * @ingroup io_interfaces
+ * @{
+ */
+
+#include <zephyr/input/input.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Common touchscreen config.
+ *
+ * This structure **must** be placed first in the driver's config structure.
+ */
+struct input_touchscreen_common_config {
+	uint32_t screen_width;
```

Comment:
Doxygen for the struct members would be good.

Reply:
Added doxygen comments